README.md

```diff
@@ -34,33 +67,31 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
 
 1. **Keeping your local copy up to date**
```

Comment:
Some of this might be overkill for here - for standard `yarn` usage details maybe refer to the docs?

Quick note here - details there. I thnk if we have too many words here, we risk people reading none of them.

Reply:
Agreed - should I bundle in these changes in with this PR or make a separate branch that improves the brevity of the readme?